logstash-core/lib/logstash/patches/clamp.rb

```diff
@@ -0,0 +1,47 @@
+require 'clamp'
+require 'logstash/environment'
+
+module Clamp
+  module Attribute
+    class Instance
+      def default_from_environment
+        # we don't want uncontrolled var injection from the environment
```

Comment:
+1.

I think it might be worth allowing ${FOO} environment values in the config yaml itself, but I'm content to wait until after the first release of this (and some user feedback) to add that.